### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by sarraKs**
Added the prerequisities I used and detailed steps for setting up the working environment.

### CODE REVIEW

**Review by jason-fox — APPROVED** [review]
LGTM - I'll copy edit this after merge

**Comment by jason-fox** [comment]
@sarraKs - copy editing is complete - please check the regenerated ReadTheDocs documentation to see if it still makes sense for you: https://fiware-tutorials.readthedocs.io/en/latest/ - most of the information has been shifted to an additional pair of appendices.

**Comment by sarraKs** [comment]
@jason-fox  - I have reviewed the regenerated ReadTheDocs documentation and found all the changes to be excellent. Thank you for your support.

**Pull request merged**
